quipucords/api/scanjob/model.py

```diff
@@ -274,3 +287,40 @@ def validate_status_change(self, target_status, valid_current_status):
                          target_status, self.status)
             return True
         return False
+
+    def create_product_status_dict(self):
+        """Construct a dictionary based on the disabled products.
```

Comment:
Feels like this method needs a better name/description. Took me a while to understand what product status was.